added reference to our code (#73)
SpecialeBajs/Rapport: article/introduction.tex

\\
Looking at papers such as \cite{Priceaware} it can be seen that propagating the influence of price on users with items as a bridge, as well as integrating category into the propagation progress increases the performance.
Since \cite{Priceaware} also utilizes a GCN our idea is to improve the performance of the LightGCN framework by utilizing price and category.

We try to extend LightGCN by adding price and categories to the embeddings and adjacency matrix.

> The code for LightGCN and the changes that we made can be seen on \footnote{Github link to LightGCN with our extension: https://github.com/SpecialeBajs/LightGCN}.

Reviewer: ```suggestion
The code for LightGCN and the changes that we made can be seen on Github \footnote{Github link to LightGCN with our extension: https://github.com/SpecialeBajs/LightGCN}.
```

> Since we got the code for \cite{Priceaware} sent to us personally through email we chose to not make it publicly available.

Reviewer: ```suggestion
Since we got the code for Price-aware Recommendation sent to us personally through email we chose to not make it publicly available as we do not have the licensing or copyright.
```

> Feel free to contact us on email if there are any question surrounding this.

Reviewer: ```suggestion
Contact us on email if there are any question surrounding this.
```

> Contact us on email if there are any question surrounding this.

Reviewer 2: ```suggestion
Contact us on email if there are any question surrounding this implementation.
```
